docs/articles_en/get-started/install-openvino.rst

```diff
@@ -48,13 +48,13 @@ Install OpenVINO™ 2024.0
 
    .. dropdown:: Distribution Comparison for OpenVINO 2024.0
 
-      ===============  ==========  ======  =========  ========  ============ ========== ==========
-       Device           Archives    PyPI    APT/YUM    Conda     Homebrew     vcpkg      Conan
-      ===============  ==========  ======  =========  ========  ============ ========== ==========
-       CPU              V           V        V         V          V           V          V
-       GPU              V           V        V         V          V           V          V
-       NPU              V          n/a      n/a       n/a        n/a         n/a        n/a
-      ===============  ==========  ======  =========  ========  ============ ========== ==========
+      ===============  ==========  ======  =========  ========  ============ ========== ========== ==========
+       Device           Archives    PyPI    APT/YUM    Conda     Homebrew     vcpkg      Conan       npm
+      ===============  ==========  ======  =========  ========  ============ ========== ========== ==========
+       CPU              V           V        V         V          V           V          V          V
+       GPU              V           V        V         V          V           V          V          V
+       NPU              V           V       n/a       n/a        n/a         n/a        n/a         V
```

Comment:
NPU is available in APT for Ubuntu 22.04 (NPU is not available on other Ubuntu versions at all, including archive)